doc/nestml_language/nestml_language_concepts.rst

```diff
@@ -645,77 +622,68 @@ NESTML supports different variants of the if-else conditional. The first example
 ::
 
     if <boolean_expression>:
-      <statements>
-    end
+        <statements>
 
 e.g.:
 
 .. code-block:: nestml
 
    if 2 < 3:
-     # <statements>
-   end
+       # <statements>
 
 The second example shows an if-else block, which executes the ``if_statements`` in case the boolean expression evaluates to true and the ``else_statements`` else.
 
 ::
 
     if <boolean_expression>:
-      <if_statements>
+        <if_statements>
     else:
-      <else_statements>
-    end
+        <else_statements>
 
 e.g.:
 
 .. code-block:: nestml
 
    if 2 < 3:
-     # <if_statements>
+       # <if_statements>
    else:
-     # <else_statements>
-   end
+       # <else_statements>
 
-In order to allow grouping a sequence of related ``if`` conditions, NESTML also supports the ``elif``-conditionals. An ``if`` condition can be followed by an arbitrary number of ``elif`` conditions. Optionally, this variant also supports the ``else`` keyword for a catch-all statement. The whole conditional always concludes with an ``end`` keyword.
+In order to allow grouping a sequence of related ``if`` conditions, NESTML also supports the ``elif``-conditionals. An ``if`` condition can be followed by an arbitrary number of ``elif`` conditions. Optionally, this variant also supports the ``else`` keyword for a catch-all statement.
 
 ::
 
     if <boolean_expression>:
-      <if_statements>
+        <if_statements>
     elif <boolean_expression>:
-      <elif_statements>
+        <elif_statements>
     else:
-      <else_statements>
-    end
+        <else_statements>
 
 e.g.:
 
 .. code-block:: nestml
 
    if 2 < 3:
-     # <if_statements>
+       # <if_statements>
    elif 4 > 6:
-     # <elif_statements>
+       # <elif_statements>
    else:
-     # <else_statements>
-   end
+       # <else_statements>
 
    if 2 < 3:
-     # <if_statements>
+       # <if_statements>
    elif 4>6:
-     # <elif_statements>
-   end
+       # <elif_statements>
 
 Conditionals can also be nested inside of each other.
 
 .. code-block:: nestml
 
    if 1 < 4:
-     # <statements>
-     if 2 < 3:
        # <statements>
-     end
-   end
+       if 2 < 3:
+           # <statements>
```

Comment:
I'd prefer these (and the ones above) to be comparisons between variables instead of numbers...